tools/Wasm.mk

```diff
@@ -42,7 +42,12 @@ define LINK_WASM
 	    $(if $(RETVAL), \
 	        $(error wasm build failed for $(PROGNAME).wasm) \
 	    ) \
-		$(call WAMR_AOT_COMPILE) \
+	    $(eval RETVAL=$(shell $(WASI_SDK_PATH)/wasm-opt -Oz -o $(BINDIR)$(DELIM)wasm$(DELIM)$(PROGNAME).wasm \
```

Comment:
* as far as i know, wasi-sdk doesn't have wasm-opt.
* see also: https://github.com/llvm/llvm-project/pull/95208

Reply:
Yes, it's from binaryen, I'll update the tool path